### ISSUE TITLE
Enable changing of optimizer options

### ISSUE THREAD

**Issue by loeiten** [acntech/fruit-classifier#36]
The current setup in `get_model` only allows for changing `epoch`, `batch_size` and `learning_rate`.

The user should be allowed to change things like optimizers, decay, loss, metric etc. This can be facilitated by the use of a factory pattern such as the one found in `fruit_classifier/models/factory.py`
